datasets/narrativeqa_manual/narrativeqa_manual.py

```diff
@@ -0,0 +1,342 @@
+                            "story_start": row[8],
+                            "story_end": row[9],
+                        }
+
+                    elif row[1] == "test":
+                        test_dict[row[0]] = {
+                            "document_id": row[0],
+                            "kind": row[2],
+                            "story_url": row[3],
+                            "story_file_size": row[4],
+                            "wiki_url": row[5],
+                            "wiki_title": row[6],
+                            "story_word_count": row[7],
+                            "story_start": row[8],
+                            "story_end": row[9],
+                        }
+
+                    elif row[1] == "valid":
+                        valid_dict[row[0]] = {
+                            "document_id": row[0],
+                            "kind": row[2],
+                            "story_url": row[3],
+                            "story_file_size": row[4],
+                            "wiki_url": row[5],
+                            "wiki_title": row[6],
+                            "story_word_count": row[7],
+                            "story_start": row[8],
+                            "story_end": row[9],
+                        }
+
+        path_to_manual_folder = os.path.abspath(os.path.expanduser(dl_manager.manual_dir))
+        onlyfiles = [f for f in listdir(path_to_manual_folder) if isfile(join(path_to_manual_folder, f))]
+
+        story_texts = {}
+        for i in onlyfiles:
+            if "content" in i:
+                with open(os.path.join(path_to_manual_folder, i), "r", encoding="utf-8", errors="ignore") as f:
+                    text = f.read()
+                    story_texts[i.split(".")[0]] = text
+
+        if not os.path.exists(path_to_manual_folder):
+            raise FileNotFoundError(
+                "{} does not exist. Make sure you insert a manual dir via `datasets.load_dataset('narrativeqa_manual', data_dir=...)` that includes the stories downloaded from the original repository. Manual download instructions: {}".format(
+                    path_to_manual_folder, self.manual_download_instructions
+                )
+            )
+
+        with open(summaries, encoding="utf-8") as csv_file:
+            csv_reader = csv.reader(
+                csv_file, quotechar='"', delimiter=",", quoting=csv.QUOTE_ALL, skipinitialspace=True
+            )
+            _ = next(csv_reader)
+
+            for id_, row in enumerate(csv_reader):
+                if row:
+                    if row[1] == "train":
+                        train_dict[row[0]]["summary"] = row[2]
+                        train_dict[row[0]]["summary_tokenized"] = row[3]
+                        try:
+                            train_dict[row[0]]["story_text"] = str(story_texts[row[0]])
+                        except KeyError:
+                            train_dict[row[0]]["story_text"] = ""
```

Comment:
```suggestion
                        train_dict[row[0]]["story_text"] = str(story_texts.get(row[0], ""))
```
I think this is simpler this way